src/doc/en/developer/doctesting.rst

```diff
@@ -1655,3 +1677,23 @@ to the doctest.
 
 Likewise, when the doctester runs into a :class:`ModuleNotFoundError`,
 the doctest fixer will automatically add a ``# needs ...`` tag.
+
+The switch ``--distribution`` can be repeated; the given distributions
+will be tested in sequence.  Using ``--distribution all`` is equivalent
+to a preset list of ``--distribution`` switches.  With the switch
+``--fixed-point``, the doctest fixer runs the given distributions until
+no more changes are made.
+
+
+Updating baseline files
+-----------------------
+
+The modularized distribution packages ``pkgs/sagemath-categories`` and
+``pkgs/sagemath-repl`` contain files ``known-test-failures*.json`` for use
+with the option ``--baseline-stats-path``, see section
+:ref:`section-doctest-auxiliary-files`.
+
+These files can be updated using the command::
+
+    [mkoeppe@localhost sage]$ ./sage --fixdoctests --no-test                        \
+                                --update-known-test-failures --distribution all
```

Comment:
Is it right that `--no-test` here?

Reply:
`--update-known-test-failures` could work without doctesting?

Reply:
Yes, without `--no-test`, it will run the doctesters first. (Not sure if that's very useful.)

Reply:
`--update-known-test-failures` without `--no-test` uses the information from previous doctests run?

Reply:
Perhaps from `timings2.json`?

Reply:
Yes; it's always coming from timings2.json

Reply:
If so, it would not hurt to write a line that says so following the above lines...

Reply:
Or I may be misunderstanding...

Reply:
I've made a change in 6aca56a; is this going in the right direction?

Reply:
Yes. That is very clear. Thanks.